lib/rubocop/cop/layout/case_indentation.rb

```diff
@@ -128,7 +128,7 @@ def detect_incorrect_style(when_node)
         def find_message(*)
           depth = indent_one_step? ? 'one step more than' : 'as deep as'
 
-          format(MSG, depth: depth, base: style)
+          super(nil, format(MSG, depth: depth, base: style))
```

Comment:
Thanks for the fixing. IMHO, there seems to be no reason for `find_message(*)` to be overridden. Can you remove this `find_message(*)` override definition and make it inline method?

```diff
diff --git a/lib/rubocop/cop/layout/case_indentation.rb b/lib/rubocop/cop/layout/case_indentation.rb
index 090bc73d7..5e6afb52f 100644
--- a/lib/rubocop/cop/layout/case_indentation.rb
+++ b/lib/rubocop/cop/layout/case_indentation.rb
@@ -105,7 +105,10 @@ module RuboCop
         end

         def incorrect_style(when_node)
-          add_offense(when_node.loc.keyword) do |corrector|
+          depth = indent_one_step? ? 'one step more than' : 'as deep as'
+          message = format(MSG, depth: depth, base: style)
+
+          add_offense(when_node.loc.keyword, message: message) do |corrector|
             detect_incorrect_style(when_node)

             whitespace = whitespace_range(when_node)
@@ -125,12 +128,6 @@ module RuboCop
           end
         end

-        def find_message(*)
-          depth = indent_one_step? ? 'one step more than' : 'as deep as'
-
-          super(nil, format(MSG, depth: depth, base: style))
-        end
-
         def base_column(case_node, base)
           case base
           when :case then case_node.location.keyword.column
```

Reply:
Definitely!

Reply:
@koic I've updated it now, thanks for the suggestion! 🙌